src/main/java/org/skriptlang/skript/bukkit/potion/elements/expressions/ExprSkriptPotionEffect.java

```diff
@@ -0,0 +1,26 @@
+package org.skriptlang.skript.bukkit.potion.elements.expressions;
+
+import ch.njol.skript.doc.NoDoc;
+import ch.njol.skript.expressions.base.EventValueExpression;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.bukkit.potion.util.SkriptPotionEffect;
+import org.skriptlang.skript.registration.SyntaxRegistry;
+
+@NoDoc
+public class ExprSkriptPotionEffect extends EventValueExpression<SkriptPotionEffect> {
+
+	public static void register(SyntaxRegistry registry) {
+		register(registry, ExprSkriptPotionEffect.class, SkriptPotionEffect.class, "[created] [potion] effect");
```

Comment:
this is too generic to not be restricted. Allow effect in a creation section, I guess, but not in generic usage. It should be `potion effect`